### PR TITLE
Updates cat trained models API spec

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR updates the cat trained model API spec.

### CODE REVIEW

**Review by delvedor — CHANGES_REQUESTED** [review]
We are already tracking most cat query parameters here: https://github.com/elastic/elasticsearch-specification/blob/b3e872e566beba79bd34a8d8b54ef857e1278f1a/specification/_spec_utils/behaviors.ts#L91-L132

There is no need to type them again unless you are changing the type :)

**Comment by szabosteve** [comment]
Thanks @delvedor for the info and for the review. I updated my PR accordingly.

File: specification/cat/ml_trained_models/CatTrainedModelsRequest.ts [hunk 1]
```diff
@@ -40,7 +40,11 @@ export interface Request extends CatRequestBase {
   query_parameters: {
     allow_no_match?: boolean
     bytes?: Bytes
+    format?: string
```
Comment (delvedor) [inline specification/cat/ml_trained_models/CatTrainedModelsRequest.ts:43]:
```suggestion
```

You can remove this one, as it's already defined here: https://github.com/elastic/elasticsearch-specification/blob/b3e872e566beba79bd34a8d8b54ef857e1278f1a/specification/_spec_utils/behaviors.ts#L91-L132 and you aren't updating the type.

File: specification/cat/ml_trained_models/CatTrainedModelsRequest.ts [hunk 2]
```diff
@@ -40,7 +40,11 @@ export interface Request extends CatRequestBase {
...
+    h?: CatTrainedModelsColumns
+    s?: CatTrainedModelsColumns
     from?: integer
     size?: integer
+    v?: boolean
```
Comment (delvedor) [inline specification/cat/ml_trained_models/CatTrainedModelsRequest.ts:48]:
```suggestion

```

Same here.

**Review by delvedor — APPROVED** [review]

**Pull request merged**